gulpfile.js

```diff
@@ -31,13 +31,7 @@ var gulp = require('gulp'),
   mqpacker = require('css-mqpacker'),
   mqsorter = require('sort-css-media-queries'),
   // Image optimization plugin
-  imagemin = require('gulp-imagemin'),
-  // Concat plugin
-  concat = require('gulp-concat'),
-  // uglify plugin
-  uglify = require('gulp-uglify'),
```

Comment:
Is it possible also removing npm dependencies?

Reply:
Yes sure. Good catch!

Reply:
No vamos a utilizar un minimo archivo custom.js como por ejemplo para interacciones del menu? O solo se utilizara puro css?

Reply:
En principio pensabamos intentaremos dejar afuera JS, cuando sea necesario se agregará.

¿Les parece bien?

Reply:
Ok